docs/Configuring_an_Application_Server.md

```diff
@@ -122,3 +122,32 @@ To update the configuration, invoke `curl` as follows.  The first stage builds t
     } | curl -X PUT http://$hs_hostname:8889/irs/<irs-uuid>/service_profiles/<service-profile-uuid>/filter_criteria --data-binary @-
 
 The subscriber will now have the desired configuration. You can confirm this by running the retrieval command again.
+
+## SIP-over-UDP configuration
+
+While it is recommended that you configure your deployment to send SIP over TCP, it is possible to create a SIP-over-UDP deployment, with certain restrictions.
+
+### Restrictions
+
+If you want to create a SIP-over-UDP deployment, it will be necessary for all of your IMS Core's SIP peers to send SIP over UDP to the IMS Core. It is not possible to set up some peers to use TCP and some to use UDP.
+
+### Configuration
+
+To enable SIP-over-UDP, you will need to set the following configuration options.
+
+In `/etc/clearwater/shared_config` set or update the field:
+
+    scscf_uri=sip:scscf.<sprout_hostname>;transport=udp
```

Comment:
I think you'll want quotes (`scscf_uri="sip:scscf.<sprout_hostname>;transport=udp"`), otherwise ; gets interpreted as Bash end-of-line and you'd get `$scscf_uri` being set to `sip:scscf.<sprout_hostname>` and `$transport` being set to `udp`. Same for scscf_node_uri below.